content/post/high-availability-with-docker.md

```diff
@@ -0,0 +1,116 @@
++++
+Categories = ["Docker", "High Availability", "Fault Tolerance", "Cluster","Raspberry Pi", "Cloud Computing"]
+Tags = ["Docker", "High Availability", "Fault Tolerance", "Cluster","Raspberry Pi", "Cloud Computing"]
+date = "2016-10-26T18:03:34+01:00"
+more_link = "yes"
+title = "Testing High Availability of Docker Swarm on a Raspberry Pi Cluster"
++++
+
+In its release in June this year, Docker announced two exciting news about the Docker Engine: First, the Engine 1.12 comes with built-in high availability features, called "Docker Swarm Mode". And second, Docker started providing official support for the ARM architecture.
+
+These two news combined beg for testing the new capabilities in reality.
+
+![SwarmClusterHA](/images/high-availability-testing/high-availability-docker-swarm.png)
+
+
+<!--more-->
+
+The "old" Docker Swarm already offered some high availability features, but the new called "Swarm Mode" goes far beyond: To me, the most important capabilities to mention are: built-in security, rolling updates and - maybe one of the most important benefits - super user-friendliness. Production ready clustering based on powerful container technology has probably never been easier before.
```

Comment:
Production ready clustering based on powerful container technology has probably never been easier before. -> Clustering based on powerful container technology has probably never been easier before.

@MathiasRenner Do we have enough experience yet to say the above?